src/main/java/com/sap/cloudfoundry/client/facade/domain/CloudRouteSummary.java

```diff
@@ -0,0 +1,86 @@
+package com.sap.cloudfoundry.client.facade.domain;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import org.immutables.value.Value;
+import org.springframework.util.StringUtils;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.sap.cloudfoundry.client.facade.Nullable;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableCloudRouteSummary.class)
+@JsonDeserialize(as = ImmutableCloudRouteSummary.class)
+public abstract class CloudRouteSummary implements Derivable<CloudRouteSummary> {
```

Comment:
Why not use directly existing CloudRoute and extend it?
What is the semantic difference?

Reply:
Because the Summary only holds the data presented in the apps/guid/summary response, so the route info there is not a full CloudEntity, as opposed to CloudRoute